### PR TITLE
Framework

### PR DESCRIPTION
**Pull request by George7h**
This PR consists of the following:

Added home page
 - shows a list of items retrieved from an API
 - Involvement API shows the item likes
 - When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.

Added comments popup.

- When the user clicks on the card, the Comments popup appears.
- When the popup loads, the webapp retrieves data from:
The selected API and shows details about the selected item.
The Involvement API to show the item comments.

### CODE REVIEW

File: src/modules/fetchDisplayComment.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const fetchAndDisplayComments = async (itemName) => {
+  const API_TOKEN = 'aQgaudShERyXiWvddmpP';
```
Comment (tugsem) [inline src/modules/fetchDisplayComment.js:2]: - [ ] The confidential information about API shouldn't be stored in GitHub. This is for security reasons. Also, after removing the keys from GitHub, please add instructions on your readme file on how to get API keys to use in their projects.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,42 @@
...
+  const shows = await fetchShows();
+  itemCounter();
+  shows.forEach(async (show) => {
+    const card = document.createElement('div');
+    card.className = 'card';
+    card.innerHTML = `
+      <img src="${show.image.medium}" alt="${show.name}">
+      <p>${show.name}</p>
+      <button class="like-button">Like</button>
+      <div class="like-count"> 0 likes</div>
```
Comment (tugsem) [inline src/index.js]: - [ ] As per the requirement, please add a 'comments' button which will take us to the popup window with the comments for the series. 
- [ ] Also, please move your like button and like counter next to the series name as in the given mockup.

yours|given
---|---
image|image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @George7h  and @Mike47ip 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️Involvement API used with other API to receive likes and comments
✔️Unıt tests for counters written
✔️App works as expected

## Required Changes ♻️
_Check the comments under the review._
-[ ] Please fix the wrong API name you have provided in the readme file to not confuse readers.
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (Mike47ip) [reply src/index.js]: Come on girl?  This is just a wireframe. The project does not require that we strictly stick to the layout of an ordinary like button??

File: src/modules/fetchDisplayComment.js (hunk 1 above)
Comment (Mike47ip) [reply src/modules/fetchDisplayComment.js:2]: Tugsem these apis are open source, there is nothing private or confidential about them. Plus how do you suggest we hide the api in the code when it is indispensable in this project?\

File: src/index.js (hunk 2 above)
Comment (tugsem) [reply src/index.js]: @Mike47ip Please be respectful. I am a reviewer, not a "girl" and I requested these changes according to the given guidelines. In a real-world experience, if you talk like this and deny all the changes in such a manner, that would not be good for you. If you think my review is unfair, then you can always ask for a second opinion. Have a good day :)

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @George7h and @Mike47ip 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**